src/main/resources/templates/admin.html

```diff
@@ -142,7 +143,7 @@ <h2 class="card-title mt-md-1 mt-0 mb-1 d-inline-block">Manage Admins</h2>
                                         </button>
                                     </div>
                                 </div>
-                                <div class="spinner-border spinner-border-sm ml-1 align-self-center"
+                                <div class="spinner-border `spinner`-border-sm ml-1 align-self-center"
```

Comment:
Correct me if I am wrong, the `` around spinner should be removed.

- [ ] Remove `` around spinner